### ISSUE TITLE
React 15.1.0 : "Cannot read property '_debugID' of undefined"

### ISSUE THREAD

**Issue by arcanis** [animatedjs/animated#35]
Hi, just a small heads up: something apparently broke in the latest React release, and my console doesn't seem to like it :)

![image](https://cloud.githubusercontent.com/assets/1037931/15655405/34126882-269d-11e6-8486-9b3cc06c7e50.png)

**Comment by browniefed** [comment]
Ah we removed an additional param from the call to mutate the CSS properties here https://github.com/animatedjs/animated/blob/master/src/targets/react-dom.js#L37 , it was used for debugging only and now it seems to be causing issues.

**Comment by Cap32** [comment]
It's been a long time after https://github.com/animatedjs/animated/pull/36 merged, but we still could not use the latest version from npm. Hope to publish a new version to npm. 🙂

**Comment by browniefed** [comment]
@vjeux

**Comment by jdmunro** [comment]
Same problem here - can we publish a newer version on npm?

**Comment by arcanis** [comment]
Ping ? 😄

In the meantime, you can fork [my repository](https://github.com/arcanis/animated) and switch your dependency target to the following in order to deploy the current master from npm:

``` json
{
    "dependencies": {
        "animated": "<your github username>/animated#master"
    }
}
```

Please don't use my repository directly in your package.json, tho - there's no guarantee I'll keep this repo up forever, so forking it is important if you want to ensure that you will always get the source code you're looking for.

**Comment by vjeux** [comment]
@arcanis hey, would you like to publish the newer version if I gave you commit access to this repo and the npm package? We don't have the time to deal with it right now unfortunately :(

**Comment by browniefed** [comment]
@vjeux I'd be happy to also publish a new package if need be.

**Comment by vjeux** [comment]
You both have access, go for it!

**Comment by browniefed** [comment]
Published a patch version 0.1.4

**Comment by arcanis** [comment]
👍 thanks @vjeux @browniefed ! Just pushed a few tags for good measure :)

**Comment by browniefed** [comment]
Ah thanks, forgot about tags

**Comment by vjeux** [comment]
Thanks all!

## 

Christopher "vjeux" Chedeau
Facebook Engineer
http://blog.vjeux.com/

**Comment by jdmunro** [comment]
Thanks, can confirm its fixed in 0.1.4!
